src/components/Routes/Routes.tsx

```diff
@@ -70,6 +75,16 @@ export function Routes() {
         }
       />
       <Route path="*" element={<PageNotFound />} />
+      {!isJuju && (
```

Comment:
As we are adding the Cross Model Search in JIMM, I thought that the user shouldn't be able to access the route when using a Juju controller.